README.md

```diff
@@ -1,2 +1,25 @@
 # Ruby-Tic-Tac-Toe
 Tic Tac Toe Game using OOP with ruby
+## Classical Tic-Tac-Toe in the terminal
+we have a 3X3 grid, with column 1-2-3 and rows a-b-c to choose which bracket we write the row a and the column 1
+so == a1 
```

Comment:
- [ ] The readme file should include some indications on how to start the game, you should have in mind that who is going to read it is someone with no prior programming knowledge (how to clone it, how to run it, etc.)